### DISCUSSION TITLE
是否应该给登陆增加个不使用代理重试的流程？

### DISCUSSION THREAD

**Discussion by LinQingYuu** [Meloong-Git/PCL#5114] [提问]
有时候可能忘了关掉代理软件就关机了，下次开机没开代理就会出现登陆不上的情况，又需要去开代理软件或者关掉代理，这似乎复杂化了操作？

**Comment by 3gf8jv4dv** [comment]
目前的行为是跟随系统代理。关于代理设置的功能可以看下 #1233。
